xbmc/dialogs/GUIDialogYesNo.cpp

```diff
@@ -121,6 +121,7 @@ bool CGUIDialogYesNo::ShowAndGetInput(const CVariant& heading,
   dialog->SetChoice(1, !yesLabel.empty() ? yesLabel : 107);
   dialog->SetChoice(2, "");
   dialog->m_bCanceled = false;
+  dialog->m_iDefaultButtonId = CONTROL_NO_BUTTON;
```

Comment:
Can you drop the hungarian notation? `m_defaultButtonId` instead of `m_iDefaultButtonId`. We don't use it anymore for the new added code

Reply:
ah ok i will do, i thought it should be maintained